packages/examples/vite-react-typescript/src/App.css

```diff
@@ -0,0 +1,12 @@
+@import url('https://fonts.googleapis.com/css2?family=Source+Sans+Pro:wght@200;300;400;600;700;900&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Bitter:wght@400;700&display=swap');
+
+@import "@department-of-veterans-affairs/css-library/dist/stylesheets/core";
+@import "@department-of-veterans-affairs/css-library/dist/stylesheets/uswds-typography";
+@import "@department-of-veterans-affairs/css-library/dist/stylesheets/utilities";
+
+@import "@department-of-veterans-affairs/component-library/dist/main";
+
+.va-h-ruled--stars {
+  background-image: url("@department-of-veterans-affairs/css-library/dist/img/stars.png");
+}
```

Comment:
🫣 Assets like this are an issue. I had to import it directly from the project.

Reply:
See this comment for more thoughts on this: https://github.com/department-of-veterans-affairs/component-library/pull/1592#discussion_r2114231868